.vitepress/theme/components/Home.vue

```diff
@@ -52,32 +51,29 @@ import SponsorsGroup from './SponsorsGroup.vue';
 
   <section id="highlights" class="vt-box-container">
     <div class="vt-box">
-      <h2>Approachable</h2>
+      <h2>Gần gũi</h2>
       <p>
-        Builds on top of standard HTML, CSS and JavaScript with intuitive
-        API and world-class documentation.
+        Xây dựng dựa trên HTML, CSS và JavaScript tiêu chuẩn với API trực quan và tài liệu tốt nhất.
       </p>
     </div>
     <div class="vt-box">
-      <h2>Performant</h2>
+      <h2>Mạnh mẽ</h2>
       <p>
-        Truly reactive, compiler-optimized rendering system that rarely
-        requires manual optimization.
+        Thật sự reactive, trình biên dịch được tối ưu để render hệ thống mà hiếm khi cần tới tối ưu hóa thủ công.
       </p>
     </div>
     <div class="vt-box">
-      <h2>Versatile</h2>
+      <h2>Đa năng</h2>
```

Comment:
```suggestion
      <h2>Linh hoạt</h2>
```

Đây là 3 từ được dùng trong tagline.